src/script/lua_api/l_env.cpp

```diff
@@ -359,6 +359,39 @@ int ModApiEnv::l_get_node_or_nil(lua_State *L)
 	return 1;
 }
 
+// bulk_get_node([pos1, pos2, ...])
+// pos = {x=num, y=num, z=num}
```

Comment:
IMO these comments are largely pointless because we have separate docs and they get out-of-date fast
```suggestion
// bulk_get_node(pos_list)
```

Reply:
This style of comment is used in many places in the `l_env.cpp` file.
So it takes send to keep it as is it or to remove/change them at all places in the file.
What solution do you prefer?

Reply:
Changing all of them is better left to a different PR.
I'd simply not add more to the pile and keep the comment shorter (as suggested).

Reply:
ok, done.